### PR TITLE
Added resolution and frame rate arg.

### PR DESCRIPTION
**Pull request by seungjunProgramming**
-x: display with (Update complete)
-y: display height (Update complete)
-z: display frame rate (Added #define for frame rate control, adding support for boot arg.)

### CODE REVIEW

**Review by pallas — CHANGES_REQUESTED** [review]
The RAOP stuff should not be moved from lib to rpiplay. For adjustable parameters, we've passed in a config block in the past. Additionally, dead code should not be left behind in a multi-line comment.

File: lib/raop_handlers.h [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #include <stdlib.h>
 #include <plist/plist.h>
 
-#define FRAMERATE 15.0
+#define FRAMERATE 15.0 //Framerate
```
Comment (pallas) [inline lib/raop_handlers.h]: Why is this a float?

Comment (seungjunProgramming) [reply lib/raop_handlers.h]: Oh
I accidentally changed the repo!
https://github.com/FD-/RPiPlay/pull/190/commits/4a31f9768016c9e5b8642f66fb74d7c776f5314c was the version that I made!
Wait a second reverting

Comment (pallas) [reply lib/raop_handlers.h]: np

**Comment by pallas** [comment]
A lot of these changes seem like incremental WIP commits. Could you rebase/squash them (e.g. all the README updates) and re-push the PR?

**Comment by seungjunProgramming** [comment]
Reverted wrong commits

**Comment by seungjunProgramming** [comment]
OK
Will Reopen PR after fixing README and etc

**Comment by pallas** [comment]
FYI, you can just force push to the merge branch to update the PR.

**Pull request closed without merging**